src/content/docs/zh-cn/start/frontend/vite.mdx

```diff
@@ -101,7 +101,7 @@ Vite 是为提供更快速，更简洁的现代 Web 项目开发体验而诞生
         host: host || false,
         port: 5173,
       },
-      // 为了使用命令行接口(CLI)为当前构建目标配置的 Tauri 环境变量
+      // 添加有关当前构建目标的额外前缀，以访问命令行接口 (CLI) 设置的 Tauri 环境变量 
```

Comment:
```suggestion
      // 用于 带着有关当前目标的信息 访问 由CLI设置的Tauri环境变量们
```
after talking with @RSS1102 on discord he suggested this. And using translate it seems more appropriate

Reply:
it is more confused than what i translated. I think I get the original meaning, it means "we can add some prefix to those environment variables which can be accessed in client code", it described in this [page](https://v2.vitejs.dev/config/#envdir)

Reply:
> it is more confused than what i translated

I see... for now we'll go with the already changed version, as @RSS1102 will try to improve the english comment. Then we'll see how it can be improved, again 😅